### PR TITLE
Fix(ColorPicker): Implement disabledAlpha (reported missing in #2528)

### PR DESCRIPTION
**Pull request by jiahao-c**
## Types of changes

- [ ] New feature
- [x] Bug fix
- [ ] Enhancement
- [ ] Documentation change
- [ ] Coding style change
- [ ] Component style change
- [ ] Refactoring
- [ ] Test cases
- [ ] Continuous integration
- [ ] Typescript definition change
- [ ] Breaking change
- [ ] Others 

## Background and context

As reported in #2528,  disabledAlpha is in the API doc but not implemented

## Solution

Implement disabledAlpha
Screenshot 2024-02-19 at 8 42 39 PM

Screenshot 2024-02-19 at 8 42 24 PM

## How is the change tested?

看了下这个组件本身没单元测试，所以就只是更新了快照。

## Changelog

| Component | Changelog(CN) | Changelog(EN) | Related issues |
| --------- | ------------- | ------------- | -------------- |
|ColorPicker| 支持 disabledAlpha | Add support for disabledAlpha  |  #2528  |

## Checklist:

- [x] Test suite passes (`npm run test`)
- [x] Provide changelog for relevant changes (e.g. bug fixes and new features) if applicable.
- [x] Changes are submitted to the appropriate branch (e.g. features should be submitted to `feature` branch and others should be submitted to `main` branch)

### CODE REVIEW

**Comment by jiahao-c** [comment]
根据以下讨论，修改了传入的初始值带透明度这种情况下的表现。

image

image

File: components/ColorPicker/input-hex.tsx [hunk 1]
```diff
@@ -62,7 +69,9 @@ export const InputHex: React.FC<InputHexProps> = ({ color, alpha, onHsvChange, o
         onPressEnter={onBlur}
         onPaste={onPaste as any}
       />
-      <InputAlpha value={alpha} onChange={onAlphaChange} />
+      {!disabledAlpha && (
+        <InputAlpha value={alpha} onChange={onAlphaChange} disabled={disabledAlpha} />
+      )}
```
Comment (flsion) [inline components/ColorPicker/input-hex.tsx:74]: `disabled` 属性应该用不到，为 true 的时候不展示 Input 了

File: components/ColorPicker/input-alpha.tsx [hunk 2]
```diff
@@ -5,9 +5,10 @@ import { ConfigContext } from '../ConfigProvider';
 interface InputAlphaProps {
   value: number;
   onChange: (value: number) => void;
+  disabled?: boolean;
```
Comment (flsion) [inline components/ColorPicker/input-alpha.tsx:8]: 这个应该不用加

Comment (jiahao-c) [reply components/ColorPicker/input-alpha.tsx]: 噢对，这个是第一版的时候加的，忘记删掉了，改好啦～

**Comment by jiahao-c** [comment]
谢谢review噢，提到的问题我改好啦

**Review by flsion — APPROVED** [review]

**Pull request merged**